tests/e2e/specs/mobile.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('Mobile experience', () => {
+  beforeEach(() => {
+    cy.viewport('iphone-6')
+
+    cy.visit('/')
+  })
+
+  it('shows the unsupported device warning', () => {
+    cy.contains('The display is too small')
+  })
+
+  describe('when the okay button is clicked', () => {
+    beforeEach(() => {
+      cy.contains('I understant').click()
```

Comment:
I understand, use anyway.
```suggestion
      cy.contains('I understand').click()
```